interface/reports/username_report.php

```diff
@@ -0,0 +1,251 @@
+<link rel="stylesheet" href="<?php echo $GLOBALS['webroot'] ?>/assets/js/fancybox-1.3.4/jquery.fancybox-1.3.4.css" type="text/css" media="screen" />
+
+<link rel="stylesheet" type="text/css" href="<?php echo $GLOBALS['webroot'] ?>/assets/js/DataTables-1.10.16/datatables.css">
+<script type="text/javascript" charset="utf8" src="<?php echo $GLOBALS['webroot'] ?>/assets/js/DataTables-1.10.16/datatables.js"></script>
+<!-- this is a 3rd party script -->
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/assets/js/datatables/extras/ColReorder/media/js/ColReorderWithResize.js"></script>
+<link rel="stylesheet" href="../../library/css/jquery.datetimepicker.css">
+<script>
+$(document).ready(function() {
+
+    if($('#details').val()) {
+        listusers();
+        console.log($("#details").val() + '= details');
+    }
+    else {
+        user_summary();
+        console.log($("#summary").val() + '= summary');
+
+    }
+
+
+});
+
+var iter=0;
+var oTable;
+
+$("#form_from_date").val();
+//Function to initiate datatables plugin
+function init_datatables()
+{
+	oTable=$('#document_table').DataTable({
+        dom: 'Bfrtip',
+        buttons: [
+            'copy', 'excel', 'pdf'
+        ],
+
+        "iDisplayLength": 100,
+        "select":true,
+        "searching":true,
+        "retrieve" : true
+	});
+
+
+
+
+
+	iter=1;
+}
+
+function refreshPage(){
+
+    window.location.reload();
+
+
+
+
+}
+
+//Function to populate document list
+function listusers()
+{
+
+
+    $.ajax({
+        type: "POST",
+        url: "../../library/ajax/username_report_ajax.php",
+        data: {
+            func:"list_all_users",
+            to_date:$("#form_to_date").val(),
+            from_date:$("#form_from_date").val()
+        },
+        beforeSend: function(){
+            $('#image').show();
+        },
+
+        success:function(data)
+        {
+            $('#users_list').html(data);
+            init_datatables();
+        },
+        complete: function(){
+            $('#image').hide();
+        }
+
+    });
+
+
+
+	iter=1;
+}
+
+function user_summary()
+{
+
+    $.ajax({
+        type: "POST",
+        url: "../../library/ajax/username_report_ajax.php",
+        data: {
+            func:"user_summary",
+            to_date: $("#form_to_date").val(),
+            from_date: $("#form_from_date").val()
+        },
+        beforeSend: function(){
+            $('#image').show();
+        },
+
+        success:function(data)
+        {
+            $('#users_list').html(data);
+            init_datatables();
+        },
+        complete: function(){
+            $('#image').hide();
+        }
+
+    });
+
+
+
+
+}
+
+
+
+
+
+</script>
+</head>
+<body class="body_top formtable">&nbsp;&nbsp;
+<form action="./username_report.php" method="post">
+<label><input value="Show Session Details" type="submit" id="details_selector" name="show_all" id="show_all"><?php ?></label>
```

Comment:
is <?php ?> needed at the end of the line. Should "Value" be translated Both of these lines

Reply:
Fixed